HippoMocks/hippomocks.h

```diff
@@ -5234,7 +5292,7 @@ TCall<void> &MockRepository::RegisterExpectDestructor(Z2 *mck, RegistrationType
 {
 	func_index idx;
 	((Z2 *)&idx)->~Z2();
-	int funcIndex = idx.lci * FUNCTION_STRIDE + FUNCTION_BASE;
+	int funcIndex = idx.lci * FUNCTION_STRIDE + FUNCTION_BASE + SUNPRO_CC_VTABLE_INDEX;
```

Comment:
SUNPRO_CC_VTABLE_INDEX is the same as FUNCTION_BASE; please reuse that variable.